src/components/document/collapsed-workspace-tab.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import ExpandIndicatorIcon from "../../assets/expand-indicator-icon.svg";
+import "./collapsed-workspace-tab.scss";
+
+interface IProps {
+  workspaceType: string;
+  onExpandWorkspace: (expand: boolean) => void;
+}
+export const CollapsedWorkspaceTab: React.FC<IProps> = ({ workspaceType, onExpandWorkspace }) => {
+  return (
+    <div className={`collapsed-workspace-tab ${workspaceType}`} onClick={()=>onExpandWorkspace(true)}>
```

Comment:
Depending on spacing preference around function definitions this could be:
`<div className={`collapsed-workspace-tab ${workspaceType}`} onClick={() => onExpandWorkspace(true)}>`

Does other CLUE code prefer the version without the spacing?  If so, leave it.